mpi-proxy-split/lower-half/mmap64.c

```diff
@@ -136,27 +131,25 @@ void
 resetMmappedList()
 {
   // Alternatively, just do:  memset((void *)mmaps, 0, sizeof(mmaps));
-  for (int i = 0; i < numRegions; i++) {
+  for (int i = 0; i < numMmapRegions; i++) {
     memset(&mmaps[i], 0, sizeof(mmaps[i]));
+    mmaps[i].addr = MAP_FAILED; // Mark those mmaps past numMmapRegions, for GDB
```

Comment:
memset should take care of setting all the variables to 0x0. We do not need to set MAP_FAILED separately. Also, the comment does not reflect the code. The loop is only till numMmapRegions, not past that.

Reply:
Did you mean to add MAP_FAILED after the for loop? The value of `i` would then be `numMmapRegions`.